internal/client/client.go

```diff
@@ -100,30 +105,33 @@ func Run(addr, fingerprint, proxyAddr string) {
 
 	sshPriv, sysinfoError := keys.GetPrivateKey()
 	if sysinfoError != nil {
-		log.Fatal("Getting private key failed: ", sysinfoError)
+		log.Println("Getting private key failed: ", sysinfoError)
 	}
 
 	l := logger.NewLog("client")
 
-	var username string
-	userInfo, sysinfoError := user.Current()
-	if sysinfoError != nil {
-		l.Warning("Couldnt get username: %s", sysinfoError.Error())
-		username = "Unknown"
-	} else {
-		username = userInfo.Username
+	if username == "" {
+		userInfo, sysinfoError := user.Current()
+		if sysinfoError != nil {
+			l.Warning("Couldnt get username: %s", sysinfoError.Error())
+		} else {
+			username = userInfo.Username
+		}
 	}
 
-	hostname, sysinfoError := os.Hostname()
-	if sysinfoError != nil {
-		hostname = "Unknown Hostname"
-		l.Warning("Couldnt get host name: %s", sysinfoError)
-	}
+	/*
+		hostname, sysinfoError := os.Hostname()
+		if sysinfoError != nil {
+			hostname = "Unknown Hostname"
+			l.Warning("Couldnt get host name: %s", sysinfoError)
+		}
+	*/
 
 	config := &ssh.ClientConfig{
-		User: fmt.Sprintf("%s.%s", username, hostname),
+		User: username, //fmt.Sprintf("%s.%s", username, hostname),
 		Auth: []ssh.AuthMethod{
 			ssh.PublicKeys(sshPriv),
+			ssh.Password(password),
```

Comment:
AS discussed in #79, the goal isnt to have clients connect to the server with a username and password. As this offers no benefit to the project, and is a weaker authentication mechanism than public/private keypair. This will be rejected if used.